test/dlc_tests/benchmark/sagemaker/pytorch/training/resources/inductor/run_inductor_timm.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+PYTHON_VERSION=$(python -c 'import sys; print(sys.version_info[0])' | tr -d "'")
+if [ "$PYTHON_VERSION" -eq 2 ]
+then
+  exit 0
+fi
+HOME_DIR=/opt/ml/output/data/test/benchmark
+BIN_DIR=${HOME_DIR}/bin
+LOG_DIR=${HOME_DIR}/logs
+
+mkdir -p ${HOME_DIR}
+mkdir -p ${BIN_DIR}
+mkdir -p ${LOG_DIR}
+
+python -c "import sys;
+assert sys.version_info < (3, 7);"
+PYTHON_VERSION_CHECK=`echo $?`
+
+set -e
+cd $BIN_DIR
+# tmp fix for numpy deprecations, PR raised: https://github.com/HewlettPackard/dlcookbook-dlbs/pull/19
+git clone --quiet --single-branch --depth 1 --branch v2.0.0 https://github.com/pytorch/pytorch.git pytorch
```

Comment:
Same as test/dlc_tests/benchmark/sagemaker/pytorch/training/resources/inductor/run_inductor_huggingface.sh. The version should not be hardcoded

Reply:
Will improve this in a new PR for inductor benchmarking